### PR TITLE
Bug 1200684: Retrieve logs from failed builds

### PR DESCRIPTION
**Pull request by kargakis**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1200684

### CODE REVIEW

**Comment by kargakis** [comment]
@bparees ptal

File: pkg/cmd/cli/cmd/startbuild.go [hunk 1]
```diff
@@ -85,15 +85,18 @@ func NewCmdStartBuild(fullName string, f *clientcmd.Factory, out io.Writer) *cob
...
-						checkErr(err)
-						defer rd.Close()
-
-						_, err = io.Copy(out, rd)
-						checkErr(err)
-						break
+					// the build is running or has failed
+					if build.Name == newBuild.Name {
+						switch build.Status {
+						case buildapi.BuildStatusRunning, buildapi.BuildStatusFailed:
```
Comment (bparees) [inline pkg/cmd/cli/cmd/startbuild.go]: couldn't it also theoretically have completed already?

Comment (kargakis) [inline pkg/cmd/cli/cmd/startbuild.go]: Well, i guess in order for a build to complete, it has to run first.

Comment (bparees) [inline pkg/cmd/cli/cmd/startbuild.go]: yes but couldn't it have transitioned through run and into complete before you do this check?  that's what's happening with failed...

Comment (kargakis) [inline pkg/cmd/cli/cmd/startbuild.go]: In theory, you are right. In reality though it's not possible to complete like that fail. Anyway, I will add the Completed status as well over there.

**Comment by bparees** [comment]
[merge]

**Comment by bparees** [comment]
[merge]

**Pull request merged**